.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: python
-python:
-  - "2.7"
-  - "3.6"
-env:
-  - AWS_DEFAULT_REGION=us-east-1
-install: 
-  - "pip install -e .[test]"
-  - "pip install coveralls"
-script: "py.test -v --cov watchbot_progress --cov-report term-missing tests/"
+cache: pip
+before_script:
+  - pip install tox coveralls
+script:  tox
 after_success:
   - coveralls || echo "intermittent coveralls failure"
+notifications:
+  on_success: change
+  on_failure: always
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.6
+      env: TOXENV=py36
```

Comment:
What is the advantage of using the build matrix? My understanding was that `tox` will handle setting up both environments.

Reply:
I think tox use python version present on the `machine` so to be able to run on python 3 and python 2 we need to different `machine` (so using travis matrix) to run in parallel